### DISCUSSION TITLE
Release date for 7?

### DISCUSSION THREAD

**Discussion by samsnori** [react-navigation/react-navigation#11805] [Q&A]
I'm about to start a new project which will take a couple of months to create and I'm wondering what version of react-navigation I should use.  I see that there has been a lot of progress with version 7, but it's still at alpha. 

Any thoughts when version 7 will be released as stable?  Should  I use 6 for now and then switch to 7? 

Maybe it's best to use 6 for now and be careful with breaking [changes](https://reactnavigation.org/docs/7.x/upgrading-from-6.x) so I can easily update to 7 once it's released.

**Comment by pfcodes** [comment]
Bump

**Comment by hraybould** [comment]
I haven't found an official release date yet, but since your original post they put a blog about [Release Candidates](https://reactnavigation.org/blog/2024/06/27/react-navigation-7.0-rc/), which you can find in the [NPM version page](https://www.npmjs.com/package/@react-navigation/native?activeTab=versions); at the moment the latest version is [7.0.0-rc.19](https://www.npmjs.com/package/@react-navigation/native/v/7.0.0-rc.19).

Obviously, you should be wary of the stability of unreleased versions, but RCs are more stable than alpha releases...
Plus it depends on if your project will be production code or a personal project (since stability concerns can be largely ignored for the latter)
